modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/LinkTag.java

```diff
@@ -328,10 +329,12 @@ protected int processStartTag() throws Exception {
 			}
 
 			if (Validator.isNotNull(_label)) {
+				String label = HtmlUtil.escape(_label);
```

Comment:
While this might have the same effect, you want to escape the contents of `jspWriter.write(`, not what you pass to `LanguageUtil.get(`. Remember, always escape as close to the context as you can.

Reply:
So, in case SF is not allowing you to chain methods, you should do:

```java
String label = LanguageUtil.get(..., _label);

jspWriter.write(HtmlUtil.escape(label));
```